### ISSUE TITLE
Translating the "Why Vue?" video

### ISSUE THREAD

**Issue by chrisvfritz** [vuejs/vuejs.org#1337]
We now have a "Why Vue?" video on the front page, which already includes closed captions (optional subtitles) for English. I'm opening this issue to coordinate translations for other languages. 

@Haeresis @ErickPetru @Jinjiang @kazupon @ludo237 @ChangJoo-Park @gbezyuk @miljan-aleksic @pi0 @thilobillerbeck Here's [the English SRT file](https://www.dropbox.com/s/futqlijoglags3h/What%20is%20Vue.js.srt?dl=0). Since it's not part of any repo, the easiest thing might just be to copy the content into a GitHub Gist and then paste the link here.

Since this is a little different than docs translations, a few tips for translating:

- You can see the duration that text appears as `00:00:02,418 --> 00:00:05,600`. That means the caption appears at 2.418 seconds into the video, then disappears at 5.6 seconds, so it appears for just over 3 seconds. Try to make sure viewers will have time to read your text in time. 🙂 Change the timing if you need to, but make sure it doesn't overlap other text.

- The length of lines is important, because lines that are too long will not be readable in a short time. Literal new lines will be respected, so when you see something like:
  ```
  00:00:02,418 --> 00:00:05,600
  more dynamic and powerful
  thanks to JavaScript.
  ```
  There will actually be a new line after `powerful`.

- Translations do not need to be literal, but note that these words are often tied directly to the timing of certain visuals in the video. That means especially for languages that require a different sentence structure (when a verb must be moved to the end of a sentence), you may want to change the phrasing and/or have part of captions start/end at different times (even if there's no actual talking at that moment).

- If you [download the video](https://fpdl.vimeocdn.com/vimeo-prod-skyfire-std-us/01/4498/9/247494684/895169613.mp4?token=1513805275-0xc96627c2c706035d6bd2ab3f836047c293985775), then you should be able to [test your SRT file in VLC media player](http://www.tomsguide.com/faq/id-1892893/srt-file-vlc-player.html). A [dedicated subtitle program](https://videoconverter.wondershare.com/subtitle/subtitle-maker.html) may also be useful, particularly if you need to change a lot of timings.

@Gregg may have other suggestions/requests for the translations, but I'll leave that to him. Thank you all so much for your continued hard work to spread the Gospel of Vue far and wide! 😄

**Comment by chrisvfritz** [comment]
Also, if anyone is translating for a culture that often prefers dubbing over subtitles and wants to try a voice over, let us know and we can talk about the best strategy for that case. @Gregg may be able to provide the source files for the video, or we might request a recording of your voice reading the subtitles you wrote in time with the video.

**Comment by pi0** [comment]
For those who get a 401 Gone error for download direct link, here is an alternative: http://www.saveitoffline.com/#https://player.vimeo.com/video/247494684

**Comment by Haeresis** [comment]
Thx @chrisvfritz for advices and pi0 for the link!

Why not include subtitles as a part of documentation repository in a `video` directory with a README.md in the same directory that explains how translate files specifically for video? Explanation in this issue is perfect. Or in a dedicated repository for all future videos, because I'm not sure a Gist was strong enough, that will allows reviews and coordination between team members in the same way as already exiting documentation?

We will start french translation ASAP ;)

**Comment by ErickPetru** [comment]
I do really agree with @Haeresis, a `video` directory would be great for now and a new repository would be even better if new videos will come in the future. Also, will we use some naming standard for the files, like `What is Vue.js.pt-BR.srt`?

**Comment by chrisvfritz** [comment]
I agree a videos repos would be useful if we do more of them, or keep iterating on this one, but we don't currently have any plans for that for either of those, so I thought this would be a good compromise in the meantime. @Haeresis

**Comment by chrisvfritz** [comment]
And actually, I don't really care where individual translators keep their SRT files. Just somewhere that allows us to download the file. 🙂

**Comment by chrisvfritz** [comment]
We also don't need a specific naming standard, as we'll take care of that when uploading to Vimeo.

**Comment by sbaidon** [comment]
Hello!

Okay so this is my shot at the Spanish Translation, of course I would appreciate if someone gave it a check mainly to improve any ortographic errors (mainly accents) and word phrasing.

PD: I really tried to make the pun at the end work, but I just couldn't, maybe someone can come up with it.  😁 

https://gist.github.com/sbaidon/0c902ace4e495972e2178b266bb84535

**Comment by chrisvfritz** [comment]
@sbaidon I just took a look and it seems very good from the perspective of a non-native speaker. 🙂 I agree it doesn't make sense to try to make the pun work - what you did instead, translating "enjoy the view" instead of "enjoy the Vue" is perfect!

I only found one item where I'm not sure if the intended meaning is captured. It might just be me, but when I read "aplicación meramente de servidor", I think of a pure API that only serves JSON with no HTML, CSS, JS that would be parsed by a browser. Maybe instead of:

```
que si tienes una aplicación 
meramente de servidor
```

What would you think of this below?

```
que si estás usando un 
framework de aplicaciones web
```

That makes me think more of frameworks like Rails or Django, which is more what's meant here.

@Gregg In reviewing the Spanish translation, I also noticed a couple small inconsistencies that also existed in the English version.

- Sometimes frameworks is capitalized and sometimes not. In both English and Spanish, I'm thinking lowercase ("frameworks" instead of "Frameworks") is probably more common. Would you say that's accurate?
- Sometimes I see "front end" or "front-end". Spanish Wikipedia uses "front-end", so maybe we should standardize on that. For English, we use "frontend" in the docs, so that might be best.

**Comment by chrisvfritz** [comment]
@Gregg I do also wonder if we should use "enjoy the view" instead of "enjoy the Vue" in our English subtitles, since the pun is really tied to a phonetic interpretation of those words. If someone who was born deaf is reading these words, I think the pun might be lost and could possibly look like a typo. What do you think?

**Comment by sbaidon** [comment]
@chrisvfritz  I think the change you propose makes sense, I will go ahead and make it.

**Comment by Gregg** [comment]
Made the changes @chrisvfritz suggested and uploaded both English and Spanish subtitles, thanks @sbaidon.  Both are live.

**Comment by ErickPetru** [comment]
Here's the brazilian portuguese version of the subtitles: https://gist.github.com/ErickPetru/bcd12d100693c5d7af9a77f1df3c2d69.

Is there any way to configure the player to auto choose this subtitles for our subdomain br.vuejs.org?

**Comment by gbezyuk** [comment]
ping @Alex-Sokolov

**Comment by Gregg** [comment]
@ErickPetru I just published your subtitles.  

Regarding having the proper subtitles showing, it looks like Vimeo implemented a feature that should automatically turn on subtitles when a viewer watches the video.  From their docs: 

"If their browser's language is different from the video’s language, and there are subtitles available for that language, they will be automatically turned on."

I just tested this in Chrome, and indeed it worked!  

Please verify you see this behavior?

**Comment by Haeresis** [comment]
Hello everyone, 
best wishes!

French translation is done @chrisvfritz : https://github.com/vuejs-fr/vuejs.org/blob/master/src/v2/video/What%20is%20Vue.js.srt

We have created this one and it is ready to be uploaded. Maybe @posva can check another time if you wants.

**Comment by Jinjiang** [comment]
@chrisvfritz How about just collecting the srt files in https://github.com/vuejs/vuejs.org/tree/master/assets for all languages. It's more maintainable imo. 🤔

**Comment by ErickPetru** [comment]
@Gregg, here the subtitles was autloaded in English even with my Chrome and my Windows configurated to Portuguese. But of course the user can change, so I think it's enough.
